docs/guides/designing-the-delta-format.md

```diff
@@ -27,7 +27,7 @@ var content = [
 ];
 ```
 
-If we want to add italics, underline, and other formats, we can add this to the main object, but it is cleaner to separate `text` from all of this so we organize formatting under one field, which we will name `attributes`.
+If we want to add italics, underline, and other formats; we can add this to the main object, but it is cleaner to separate `text` from all of this so we organize formatting under one field, which we will name `attributes`.
```

Comment:
Fourth item in the first link
<img width="515" alt="screen shot 2016-10-23 at 19 29 41" src="https://cloud.githubusercontent.com/assets/2107110/19630346/1d1fef3c-9957-11e6-9fc6-92028c7a3853.png">

However, after further research, it seems the initial `if` makes it invalid as you well have said: it is not an independent clause... my bad.
Changing it.